src/components/sliders/attributes.js

```diff
@@ -20,7 +20,7 @@ var stepsAttrs = {
 
     method: {
         valType: 'enumerated',
-        values: ['restyle', 'relayout', 'animate', 'update'],
+        values: ['restyle', 'relayout', 'animate', 'update', 'skip'],
```

Comment:
A little note on `'skip'` in the description would've been nice. We can get to that later though.